ogr/services/pagure.py

```diff
@@ -50,6 +50,10 @@ def __init__(
 
         self.header = {"Authorization": "token " + self._token} if self._token else {}
 
+    def __str__(self) -> str:
+        return (
+            f"Pagure-Service"
```

Comment:
What about adding also `instance_url`. It would be useful to see it...

Reply:
Yes. It will be. I will make the changes. Thanks for your valuable feedback !